### ISSUE TITLE
credentials dont work in demo site

### ISSUE THREAD

**Issue by wileymimi** [kmanjari/hrms#41]
the creditials you provided do not work in the demo site

**Comment by chinyati** [comment]
I agree, can you please provide us the correct login details for demo site

**Comment by khanof89** [comment]
credentials for demo site: 
test@demo.com
123456

I will update the readme file really soon

**Comment by sthaiya** [comment]
Credentials dont work

**Comment by khanof89** [comment]
we have allowed the user to change the credentials on demo site hence some people who are testing the functionality do change the password, will add a cron to revert it back disallow changing the credentials. Temporarily changed the password to 123456

**Comment by sthaiya** [comment]
Thanks I can now login

**Comment by amitdhongde** [comment]
The credentials for the demo site doesn't work. Can you assist?

**Comment by abhiburk** [comment]
Same with me.The credentials for the demo site doesn't work.

**Comment by gokuldhas** [comment]
Demo credentials not working

**Comment by Franogales** [comment]
this issue is still not working, I guess if you add an extra validation for this user?

**Comment by albisanti** [comment]
What about removing the possibility of changing the password? Just comment it on the demo so this issue will not exists anymore

**Comment by albisanti** [comment]
I'll probably work on this.

I'll make a pull request when everything is working

**Comment by KhalilMohammad** [comment]
Demo credentials not working.

**Comment by khanof89** [comment]
Demo credentials are working now, made some changes so that credentials cannot be changed on demo mode
